integrations/transportation/tfl.js

```diff
@@ -124,9 +124,28 @@ async function collect(state) {
       throw new HTTPError(e);
     });
 
+  // Update contactless cards
+  const contactlessCardResponse = await fetch(`${BASE_PATH}/Contactless/Cards`, {
+    method: 'get',
+    headers: {
+      'x-zumo-auth': newState.accessToken,
+    },
+  })
+    .then(response => response.json())
+    .catch((e) => {
```

Comment:
Just a suggestion:
I feel it's simpler if you wrap the whole thing in a try catch:
```js
try {
  const contactlessCardResponse = await fetch(`${BASE_PATH}/Contactless/Cards`, {
      method: 'get',
      headers: {
        'x-zumo-auth': newState.accessToken,
      },
    });
  const contactlessCardResponseObject = await contactlessCardResponse.json();
} catch (e) {
  throw new HTTPError(e);
}
```

Reply:
I've wrapped the fetch in its own function to make this cleaner